packages/atomic/src/pages/v1.html

```diff
@@ -231,12 +219,25 @@
                   font-weight: bold;
                   margin-right: 0.25rem;
                 }
+
+                atomic-result-section-badges {
+                  display: flex;
+                }
```

Comment:
Without making this a flexbox, badges are pushed down about 2 pixels. I couldn't find any border, padding or margin anywhere to explain or cancel that behaviour.